src/Http/Http/src/Internal/StreamResponseBodyFeature.cs

```diff
@@ -0,0 +1,118 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.IO.Pipelines;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http.Features;
+
+namespace Microsoft.AspNetCore.Http
+{
+    public class StreamResponseBodyFeature : IHttpResponseBodyFeature
```

Comment:
Make it internal or move it to a public namespace?

Reply:
It is in a public namespace, but I can move it out of the Internal folder.

Reply:
Do you think this should be called DefaultResponseBodyFeature?

Reply:
> Do you think this should be called DefaultResponseBodyFeature?

I'd rather not. I expect there to be other variants like PipeResponseBodyFeature that takes the pipe writer as the primary input.